### PR TITLE
Docs: mob review

### PR DESCRIPTION
**Pull request by estherthetesteramido**
#### 📲 What

Mob review session on all things documentation.

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

**Comment by naren-bellala** [comment]
This is really good. we should add it to our other Docs folder too!!!

File: .gitignore [hunk 1]
```diff
@@ -14,5 +14,3 @@
 website/build/
 website/i18n
 website/node_modules
-website/package-lock.json
-yarn.lock
```
Comment (dnitsch) [inline .gitignore:18]: no new line?

File: docs/getting_started_dev.md [hunk 2]
```diff
@@ -6,24 +6,24 @@ sidebar_label: Dev - Using the repo to build an application
...
-  $ npm run build
-  $ npm run build:watch
-  $ npm npm run start
+  npm install
+  npm run build
+  npm run build:watch
+  npm npm run start
   ```
 
 All of our packages, including the `@amidostacks/scaffolding-api` are available on [npm](https://www.npmjs.com/package/@amidostacks/scaffolding-cli).
```
Comment (dnitsch) [inline docs/getting_started_dev.md:25]: `@amidostacks/scaffolding-api` ==> `...-cli`?

File: .gitignore (hunk 1 above)
Comment (estherthetesteramido) [reply .gitignore:18]: There is in my IDE...

**Pull request merged**